REQUIRE

```diff
@@ -1,3 +1,3 @@
-julia 0.4
-DataStructures 0.3.8
-Compat 0.7.16
+julia 0.5
+DataStructures 0.6.0
```

Comment:
Hi, PriorityQueue moved to DataStructures in 0.6 right -- not sure I understand what you mean?